### ISSUE TITLE
TimeScaleLabels overlap each other for bigger data range

### ISSUE THREAD

**Issue by jbalcerzak** [dlemmermann/CalendarFX#94]
![dayRangeAs7days](https://user-images.githubusercontent.com/65945003/82875309-8749cd80-9f37-11ea-8d6a-ca64d52f7d47.png)

If number of visible hours is to big then labels start to overlap each other as shown in attached image. I would like to hide some of them, but I could not find anything that could do that.
Is there any option, that could achive that, if not, could be it implemented?

Regards,
Jan Balcerzak, PSI

**Comment by dlemmermann** [comment]
This feature is not available, yet, but it can be added to TimeScaleViewSkin. Currently this feature is out-of-scope of our current agreement, so you have the following options: 

a) you pay the second half of the invoice, which will make me very happy and then I will be inclined to support this feature free of charge 
b) PSI enters into the maintenance contract we discussed for a while now or 
c) you dig into the code and check for overlapping labels and submit a pull request, which I will gladly merge

To clarify: you ordered a specific set of new functionality, which I implemented. The timescale view always had this limitation, this could have been known to you as everything is out there in the open and you had ample opportunity / time to verify the calendar framework. None of your specs / mock-ups indicated that you want to display such small hours.

A slightly annoyed ….

Dirk
